spec/models/course_spec.rb

```diff
@@ -94,6 +94,66 @@
         end
       end
 
+      describe '#current_level_threshold' do
+        subject do
+          -> (course, experience_points) { course.current_level_threshold experience_points }
```

Comment:
subjects should not be lambdas, subjects should be directly usable.

What I usually do is:
```ruby
subject { build(:some_factory, *factory_traits) }
let(:factory_traits) { nil }

context 'some context' do
  let(:factory_traits) { :some_trait }
  it { is_expected.to be_truthy }
end
```